hudi-client/src/main/java/org/apache/hudi/HoodieWriteClient.java

```diff
@@ -510,21 +515,21 @@ private Partitioner getPartitioner(HoodieTable table, boolean isUpsert, Workload
   /**
    * Commit changes performed at the given commitTime marker.
    */
-  public boolean commit(String commitTime, JavaRDD<WriteStatus> writeStatuses) {
-    return commit(commitTime, writeStatuses, Option.empty());
+  public boolean commit(String commitTime, JavaRDD<WriteStatus> writeStatuses, WriteOperationType operationType) {
```

Comment:
As only one hudi write operation is outstanding at a time, can you cache the last operation type in instance variables within HoodieWriteClient object so that users don't need to explicitly pass them in this commit() call